### ISSUE TITLE
NTR: Piloerection, excessive (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8008]
New term request Piloerection, excessive (samant_110131002411): Cutis anserina; excessive goose bumps  UMLS: 0031924

**Comment by pnrobinson** [comment]
@MickeySegal
I believe that cutis anserina means simply "goose bumps" and is not abnormal.
What is the context?

**Comment by MickeySegal** [comment]
This occurs to an excessive degree in conditions such as opioid withdrawal and sympathomimetic drug toxicity.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000958
- label: Excessive piloerection
- parents: [Abnormality of skin adnexa physiology (HP:0025276)](https://hpo.jax.org/app/browse/term/HP:0025276)
- pmid: PMID:30252268
- def: Abnormally increased degree of goose bump formation, that is, of involuntary formation of bumps on a person's skin at the base of body hairs.
- comment: Excessive piloerection can be a feature of opioid withdrawal.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8008
- synonyms: Excessive goose bumps(NONE)|Excessive cutis anserina(NONE)
